### PR TITLE
Validate Inputs for activate/track/getVariation

### PR DESCRIPTION
**Pull request by oakbani**

### CODE REVIEW

File: tests/TestData.php [hunk 1]
```diff
@@ -792,6 +795,11 @@ public function sendImpressionEvent($experimentKey, $variationKey, $userId, $att
     {
         parent::sendImpressionEvent($experimentKey, $variationKey, $userId, $attributes);
     }
+
+    public function validateInputs(array $values, $logLevel = Logger::ERROR)
```
Comment (oakbani) [inline tests/TestData.php]: Wrapper public method to unit test

**Comment by mikeng13** [comment]
build

**Comment by mikeng13** [comment]
build

**Comment by mikeng13** [comment]
build

**Comment by mikeng13** [comment]
Expected failing E2E test.

**Pull request merged**